app.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env sh
+#
+# This script is run by OpenShift's s2i. Here we guarantee that we run desired
+# command and debug level
+#
+
+set -o nounset
+set -o errexit
+set -o errtrace
+set -o pipefail
+trap 'echo "Aborting due to errexit on line $LINENO. Exit code: $?" >&2' ERR
+
+DEBUG_LEVEL=${DEBUG_LEVEL:-0}
+DEBUG_FLAGS=""
+
+if [ "$DEBUG_LEVEL" -eq 1 ]; then
+    DEBUG_FLAGS="--debug --loglevel debug"
+fi
+
+exec faust ${DEBUG_FLAGS} -A producer cli
```

Comment:
@fridex I feel like this is not the right way to go, WDTY?

Reply:
Yes - I haven't tried it but it looks good to me